app/assets/javascripts/application.js.coffee

```diff
@@ -0,0 +1,23 @@
+// Draw and forfeit actions
+
+$ ->
+  window.broadcast_channel.bind 'draw_forfeit', (event) ->
+    $('#message').html(event.message)
+
+// Show default buttons
```

Comment:
`//` is not a comment in coffee i dont think. Did get an error for this. Changed it to # and error went away.